tests/test_git.py

```diff
@@ -607,3 +607,29 @@ def test_checkout_subdir(tmp_dir: TmpDir, scm: Git, git: Git):
     with (tmp_dir / "dir").chdir():
         git.checkout(rev)
         assert not (tmp_dir / "dir" / "bar").exists()
+
+
+def test_dulwich_describe(tmp_dir: TmpDir, scm: Git):
```

Comment:
```suggestion
@pytest.mark.skip_git_backend("pygit2", "gitpython")
def test_describe(tmp_dir: TmpDir, scm: Git):
```

Reply:
also, please update the test to use the `git: Git` fixture, and use `git.describe` instead of `scm.describe`

It is safe to use `scm` for calls that set up the git repo and git state in the test function, but the `git` fixture specifically forces using the particular backend we want in the specific call/feature/etc we are testing.